modules/file-rotation/file-rotation.c

```diff
@@ -24,11 +24,34 @@
 #include "file-rotation.h"
 #include "modules/affile/file-signals.h"
 #include "driver.h"
+#include "time.h"
 
 static void
 _slot_file_rotation(FileRotationPlugin *self, FileFlushSignalData *data)
 {
   msg_trace("File rotation signal received", evt_tag_str("filename", data->filename));
+
+  if(data != NULL)
+    {
+      if(data->size >= self->size)
+        {
+          msg_debug("File size reached the limit", evt_tag_str("filename", data->filename));
```

Comment:
I think we could produce a `msg_info()` here: "File size limit reached, rotating file".
We could add the original filename, the new filename, and the size to the info message as well.

Reply:
Updated Accordingly